cargo-apk/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.6.0 (2021-05-06)
```

Comment:
Looks like I've botched the pattern in the previous release. This should be `# Unreleased`.

At some point we'll have to investigate `release.toml` and the automated deploy process to take take care of updating this: https://github.com/sunng87/cargo-release/blob/master/docs/faq.md#maintaining-changelog